pkg/sdk/internal/client/client_test.go

```diff
@@ -29,17 +28,3 @@ func TestNewClientWithoutInstrumentedSQL(t *testing.T) {
 		assert.Contains(t, sql.Drivers(), "snowflake")
 	})
 }
-
-func TestNewClientWithDebugLoggingSetFromEnv(t *testing.T) {
```

Comment:
let's skip this one for now and add it to the issue I asked about in a separate comment

Reply:
This can be reverted in the next PR.